### ISSUE TITLE
pytorch-pretrained-bert now updated to pytorch-transformers

### ISSUE THREAD

**Issue by desireevl** [lonePatient/Bert-Multi-Label-Text-Classification#14]
The pytorch-pretrained-bert packaged being used has recently updated to pytorch-transformers. This means that import statements needed to be updated as well as some minor changes such as the way the optimiser and scheduler are written as well as the model now returning a tuple. The major changes are found [here](https://github.com/huggingface/pytorch-transformers#Migrating-from-pytorch-pretrained-bert-to-pytorch-transformers). 

I'm currently doing my best to try update this code for it to work with the new changes.

**Comment by lonePatient** [comment]
I update to `pytorch-transoformers`, and add `xlnet` model ,Lookahead optimizer RAdam optimizer .....
